src/components/views/elements/Slider.tsx

```diff
@@ -83,18 +89,73 @@ export default class Slider extends React.Component<IProps> {
 
         let selection = null;
 
+        const onDrag = (e: React.DragEvent) => {
+            //  console.log(e.target.parentNode.parentNode)
+
+            const target = e.target as HTMLElement;
+            if (!target.parentNode) return null;
+            const parent = target.parentNode as HTMLElement;
+            if (!parent.parentNode) return null;
+            const slider = parent.parentNode as HTMLElement;
```

Comment:
I believe we should rely on react refs instead of traversing the DOM.
The advantage I see is that we explicitely declare refs on the DOM element we wish to track. 
With your approach, we rely on the fact that the DOM structure, and that is subject to change in the future